docs/events/eboard-meetings.rst

```diff
@@ -0,0 +1,49 @@
+########
+Eboard Meetings
+########
+
+This page documents RITlug eboard meetings.
```

Comment:
"This page documents RITlug executive board (eboard) meetings.

Reply:
I'm all for clarifying terms like this at the start of a document. For all we know this could be the first page a new RITlug member reads. I spent part of my first year at RIT thinking 'eboard' stood for 'electronic board' - which caused a mild amount of confusion.